enterprise/backend/src/metabase_enterprise/audit_db.clj

```diff
@@ -129,7 +132,11 @@
   (cond
     zip-resource
     (do (log/info "Unzipping instance_analytics to " plugin-dir "...")
-        (u.files/unzip-file analytics-zip-resource plugin-dir)
+        (u.files/unzip-file analytics-zip-resource
+                            (fn [entry-name]
+                              (str/replace entry-name
+                                           "resources/instance_analytics/"
+                                           "plugins/instance_analytics/")))
```

Comment:
probably shouldn't be hard-coding `plugins` here either

Reply:
yeah, even the `"resources/instance_analytics/"` doesn't feel quite right. I'll go figure out a better abstraction tomorrow.